opal/corelib/enumerable.rb

```diff
@@ -177,6 +177,10 @@ def count(object = undefined, &block)
   end
 
   def cycle(n = nil, &block)
+    if `arguments.length > 1`
```

Comment:
re: arity checking being turned off in the Rake task, not that I know of, i still see '-A' in the args when the mspec_opal_node and mspec_rubyspec_node Rake tasks are run.